src/scheduler/limits.c

```diff
@@ -1203,6 +1203,9 @@ check_server_max_user_run(server_info *si, queue_info *qi, resource_resv *rr,
 	if ((si == NULL) || (user == NULL) || (sc == NULL))
 		return (SCHD_ERROR);
 
+	if (si->has_user_limit != 1)
```

Comment:
This is a bit, use a !, don't check if it is not equal to 1. (and other places in the PR)